### ISSUE TITLE
ScriptError in showing map

### ISSUE THREAD

**Issue by nkentNTIA** [pbugnion/gmaps#122]
Trying to run the example stuff from the docs, and I can only get through the following before it crashes:

```
import os
import gmaps

from gmaps import datasets

gmaps.configure(api_key=os.environ["GOOGLE_API_KEY"])
```

```
earthquake_df = datasets.load_dataset_as_df("earthquakes")
earthquake_df.head()
```

```
locations = earthquake_df[["latitude", "longitude"]]
weights = earthquake_df["magnitude"]
m = gmaps.Map()
m.add_layer(gmaps.heatmap_layer(locations, weights=weights))
m
```

The `m` line produces this error:
```
Could not create model:
Model name	PlainmapModel
Model module	jupyter-gmaps
Model module version	*
Script error for "jupyter-gmaps" http://requirejs.org/docs/errors.html#scripterror
```

I'm using the Anaconda distro and I've tried running this in both Py2.7 and Py3.6. Any ideas on what the issue might be here?

**Comment by pbugnion** [comment]
Thanks for the bug report.

Interesting -- I managed to reproduce this briefly by:

1. Opening a notebook
2. Installing gmaps with pip install, but not enabling the jupyter JS extension
3. Running through the code examples (at least the `import gmaps` part, which injects the dependency on the Google maps javascript API). This crashes with a slightly different error message (something of the ilk 'Model module version undefined')
4. Remembering to enable the juptyer JS extension with `jupyter nbextension enable --py --sys-prefix gmaps`

I then reproduced the error. 

I fixed it by either restarting the notebook server and closing the notebook tab, or restarting the kernel and refreshing the browser page.

Let me know if this fixes the error. If not, it would be useful to see the output from the browser console.
